package.json

```diff
@@ -2,9 +2,11 @@
   "name": "outline-client",
   "productName": "Outline",
   "scripts": {
+    "action": "bash ./scripts/run_action.sh",
+    "action:help": "npm run action",
+    "action:list": "npm run action",
     "clean": "rm -rf build www node_modules platforms/* plugins/*",
-    "do": "bash ./scripts/do_action.sh",
-    "actions": "find . -name '*_action.sh' | sed -E 's:./src/(.*)_action.sh:\\1:' | grep -v 'scripts/do_action'"
+    "lint:ts": "tslint 'src/**/*.ts'"
```

Comment:
I think we should have a `"lint"` action.  If you want, it can be an alias to `lint:ts`.

Reply:
yes, love it